heron/cli/src/python/BUILD

```diff
@@ -17,7 +17,8 @@ pex_library(
         "submit.py",
         "utils.py",
         "version.py",
-    ], 
+        "cli_helper.py",
```

Comment:
Put it in alphabetical order.

In other places, we used globs so that we don't need to add every file manually.

```
srcs = glob(
        ["**/*.py"],
        exclude = ["main.py"],
    ),
```

I won't mind doing that here either.